plugins/providers/gcloudiam/config.go

```diff
@@ -147,8 +147,8 @@ func (c *Config) validateResourceConfig(resource *domain.ResourceConfig) error {
 	}
 
 	if resource.Roles == nil || len(resource.Roles) == 0 {
```

Comment:
this is redundant, afaik we just need to check if `len(resource.Roles) == 0`
[reference](https://cs.opensource.google/go/go/+/refs/tags/go1.18.4:src/builtin/builtin.go;l=161)

Reply:
Done